translation.rst

```diff
@@ -1289,6 +1289,132 @@ adapted to the format required by GitHub, but you can force that format too:
 
         The ``yaml-lint`` binary was introduced in Symfony 5.1.
 
+Pseudo-localization translator
+------------------------------
+
+The following image shows the main menu of the interface of a popular Internet
```

Comment:
I would add the version added here at first paragraph of this bloc
and also add a tip or notice that this is meant for development to help
wdyt?
appart from that, nice to add this :)

Reply:
Done, thanks :)